backend/check_persona_system_prompts.py

```
@@ -0,0 +1,107 @@
+        print()
+        
+        for persona in personas:
+            print(f"📋 Persona: {persona.name} (ID: {persona.id})")
+            print(f"   Role: {persona.role}")
+            print(f"   System Prompt Length: {len(persona.system_prompt) if persona.system_prompt else 0}")
+            
+            if persona.system_prompt:
+                # Check for trigger words
+                system_prompt = persona.system_prompt.lower()
+                
+                print(f"   🔍 Trigger Analysis:")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)